packages/qwik/src/optimizer/src/plugins/vite.ts

```diff
@@ -1175,7 +1175,8 @@ export function convertManifestToBundleGraph(manifest: QwikManifest): QwikBundle
         // external dependency
         continue;
       }
-      if (dep.isTask) {
+      // prevent importing all segments chunks
+      if ((dep.isTask || dep.isEntry) && !dep.isPlan) {
```

Comment:
@wmertens just FYI, the `!dep.isPlan` check doesn't change anything in terms of actual prefetching on either qwik-ui webiste or qwik docs website. It is unnecessary imo but I'm fine with keeping it ^^

Reply:
Hmm how is that possible? If we're fetching the full dynamic graph, doesn't the plan then cause the entire application to fetch?

Reply:
I think the qwik SW never imports the qwik-city plan, and the qwik-city SW uses it for the LinkBundles maybe? I'm not sure about that part.